docs/openstack/heat_adoption.md

```diff
@@ -0,0 +1,158 @@
+# Heat adoption
+
+Adopting Heat means that an existing `OpenStackControlPlane` CR, where Heat
+is supposed to be disabled, should be patched to start the service with the
+configuration parameters provided by the source environment.
+
+After the adoption process has been completed, a user can expect that they
+will then have CR's for `Heat`, `HeatAPI`, `HeatEngine` and `HeatCFNAPI`.
+Additionally, a user should have endpoints created within Keystone to facilitate
+the above mentioned servies.
+
+This guide also assumes that:
+
+1. A `TripleO` environment (the source Cloud) is running on one side;
+2. A OpenShift environment is running on the other side.
+
+## Prerequisites
+
+* Previous Adoption steps completed. Notably, MariaDB and Keystone
+  should be already adopted.
+* In addition, if your existing Heat stacks contain resources from other services
+  such as Neutron, Nova, Swift, etc. Those services should be adopted first before
+  trying to adopt Heat.
+
+## Procedure - Heat adoption
+
+As already done for [Keystone](https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/keystone_adoption.md), the Heat Adoption follows a similar pattern.
+
+Patch the `osp-secret` to update the `HeatAuthEncryptionKey` password. This needs
```

Comment:
I think both the Heat service password and auth encryption key should be added to:

https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/docs/openstack/backend_services_deployment.md

and to

https://github.com/openstack-k8s-operators/data-plane-adoption/blob/b4019f55eb29a8e6c8efaa8240a7cdf31de880cf/tests/roles/backend_services/tasks/main.yaml#L14

Reply:
This was done, just didn't resolve the conversation. Resolving.